modules/sandboxed-containers-deleting-kataconfig-cli.adoc

```diff
@@ -32,6 +43,11 @@ $ oc delete crd kataconfigs.kataconfiguration.openshift.io
 
 The {sandboxed-containers-operator} removes all resources that were initially created to enable the runtime on your cluster.
 
+[IMPORTANT]
+====
+During deletion, the Command Line Interface freezes until all worker nodes reboot. Wait for the process to complete before performing the verification or continuing to the next procedure.
+====
+
```

Comment:
Small clarification : the command that freezes is `oc delete kataconfig <KataConfig_CR_Name>` so maybe this note should go between step 1 and step 2.  @tbuskey do you agree ?

Reply:
@gkurz Yes, that's the command